src/app/mat-select-search/mat-select-search.component.html

```diff
@@ -1,8 +1,11 @@
 
 <input matInput class="mat-select-search-input mat-select-search-hidden"/>
 
-<div class="mat-select-search-inner"
-     [ngClass]="{'mat-select-search-inner-multiple': matSelect.multiple}">
+<!-- Note: the  mat-datepicker-content mat-header-row are needed to inherit the material theme colors, see PR #22 -->
+<div
+      #innerSelectSearch
+      class="mat-select-search-inner mat-typography mat-datepicker-content mat-header-row"
+      [ngClass]="{'mat-select-search-inner-multiple': matSelect.multiple}">
```

Comment:
with material `5.2.3`, the `.mat-header-row` class does not seem to exist in e.g. the deeppurple theme, resulting in a black line, see
![image](https://user-images.githubusercontent.com/5589029/39625158-8ddb99d6-4f9c-11e8-9cfd-6f3f7504eac6.png)

Reply:
dammit... Well i think i should do the mixins implementation then...

Reply:
I found why it doesn't work: it is limited to `th.mat-header-row` elements, see
https://github.com/angular/material2/blob/master/src/lib/table/_table-theme.scss#L14

Reply:
it works with `.mat-tab-header`

Reply:
okay nice!